src/tokens/polygonTokens.json

```diff
@@ -20518,5 +20518,24 @@
                 "website": "https://peercoin.net/"
             }
         }
+    },
+    {
+        "chainId": 137,
+        "name": "Kapital DAO Token",
+        "symbol": "KAP",
+        "decimals": 18,
+        "address": "0x8e26032650a389BF9cf16C17C4cd605742f1ef8D",
```

Comment:
Looks like the root token is mapped to address 0xc27158bB8E08899d38765752F91d7d778e16AB8e on polygon PoS chain. Please review the addition.

Reply:
Yes, that was the original bridged token, but after a lengthy process we've rebridged with a custom bridge to customize the token name and symbol.

FxERC20RootTunnel - 0x89748a6Aa14d7dC1D3a669b46891e1fD44Ff59f6
FxERC20ChildTunnel - 0xD06057dff23f728483dC8C4a48fa94bF84788291

Reply:
Got it. I'm assuming you have used a custom fx bridge implementation here. In that case can you please add the "rootTunnelAddress" of your custom bridge? [Here](https://github.com/maticnetwork/polygon-token-list/blob/584550b7629b1080f30ce1108832a97c0a0a4086/src/tokens/polygonTokens.json#L20458) is an example you can follow. Also in the tags instead of "pos" please use "fx" as bridgeType.

We'll add documentation for these custom cases as well into the readme file soon, looks like we missed it. Thank you.

Reply:
That's correct. Thank you so much for all the help. I've applied your requested changes.